worker.py

```diff
@@ -141,6 +143,10 @@ def run(self):
                 logger.warning('Malformed response received!')
                 self.error_code = 'RESTART'
                 self.restart()
+            except BannedAccount:
+                self.error_code = 'BANNED'
+                self.running = False
+                time.sleep(999999999999)
```

Comment:
Ugh! Just return here, thread (or coroutine) will finish and won't be restarted, as `running` is set to false.